v2/manager.go

```diff
@@ -337,6 +337,14 @@ func (c *Manager) AddProc(pid uint64) error {
 }
 
 func (c *Manager) Delete() error {
+	// kernel prevents cgroups with running process from being removed, check the tree is empty
+	processes, err := c.Procs(true)
+	if err != nil {
+		return err
+	}
+	if len(processes) > 0 {
+		return fmt.Errorf("cgroups: unable to remove path %q: still contains running processes", c.path)
```

Comment:
Is it possible to have a similar message on cgroup.go?

Reply:
The v1 Delete just accumulates a list of subsystems that couldn't be removed and doesn't report the error. I could make the append more like:

`errs = append(errs, fmt.Sprintf("%s (contains running processes)", string(s.Name()))`

be a solution? Then the resulting error would be akin to

`cgroups: unable to remove paths memory (contains running processes), cpu (contains running processes)`

Reply:
sounds good to me!

Reply:
Apologies for not getting back to this sooner, I added the "contains running process" reason to the errors list in cgroup.go